CDCgov/prime-simplereport
# Add patient gender identity to bulk upload

# BACKEND PULL REQUEST

## Related Issue

- Resolves #8224 

## Changes Proposed

- Adds `patient_gender_identity` column as an optional field on the bulk results uploader

## Testing

- Deployed [a branch with an extra log statement](https://github.com/CDCgov/prime-simplereport/tree/mike/log-8224-csv-patient-gender-identity) to dev5
- Use this example csv file to upload results on dev5 (you can also clone the branch and test locally)
    - [test-results-upload-patient-gender-identity.csv](https://github.com/user-attachments/files/17653956/test-results-upload-patient-gender-identity.csv)
 - Check the logs for `"Gender identity"},"subject":{"reference":"Patient/` to confirm that the patient's gender identity is mapped in the uploaded FHIR bundle
